### PR TITLE
Add orientation information to accelerometer spec

### PR DESCRIPTION
**Pull request by jamesadevine**

### CODE REVIEW

File: services/accelerometer.md [hunk 1]
```diff
@@ -7,6 +7,18 @@
...
+
+An accelerometer module should translate acceleration values as follows:
+
+| Orientation           	| X value 	| Y value  	| Z value 	|
+|-----------------------	|---------	|----------	|---------	|
+| Module lying flat     	| 0       	| 0        	| -1000   	|
+| Module on "left" edge   	| -1000   	| 0        	| 0       	|
+| Module on "bottom" edge 	| 0       	| 1000     	| 0       	|
+
+We recomment an orientation marking on the PCB so that users can mount modules without having to experiment with the device.
```
Comment (pelikhan) [inline services/accelerometer.md:20]: this would be a brilliant example of test.

Comment (jamesadevine) [reply services/accelerometer.md]: Agreed! (cc @tballmsft)

File: services/accelerometer.md [hunk 2]
```diff
@@ -7,6 +7,18 @@
...
+
+An accelerometer module should translate acceleration values as follows:
+
+| Orientation           	| X value 	| Y value  	| Z value 	|
+|-----------------------	|---------	|----------	|---------	|
+| Module lying flat     	| 0       	| 0        	| -1000   	|
+| Module on "left" edge   	| -1000   	| 0        	| 0       	|
+| Module on "bottom" edge 	| 0       	| 1000     	| 0       	|
+
+We recommend an orientation marking on the PCB so that users can mount modules without having to experiment with the device.
```
Comment (mmoskal) [inline services/accelerometer.md:20]: should be -1/1 not -1000/1000 (unit is g)

I would also say "module standing upright on "left" edge" to be more precise?

Also some wording like: "left/bottom can be determined by assuming text on silk runs left-to-right"

Comment (jamesadevine) [reply services/accelerometer.md:20]: Thanks! will edit

**Review by pelikhan — APPROVED** [review]
evenutally tests for this would be gold.

**Review by mmoskal — APPROVED** [review]
don't forget to run make and check in before merging

**Pull request merged**